recipes-browser/cog/cog-meson.inc

```diff
@@ -0,0 +1,22 @@
+inherit meson
+
+PACKAGECONFIG[dbus] = "-Dcog_dbus_control=system -Dcog_dbus_system_owner=${COG_DBUS_OWN_USER},-Dcog_dbus_control=user"
+PACKAGECONFIG[drm] = ",,wpebackend-fdo libdrm virtual/libgbm libinput"
+PACKAGECONFIG[headless] = ",,wpebackend-fdo"
+PACKAGECONFIG[soup2] = "-Dsoup2=true,-Dsoup2=false,libsoup-2.4"
```

Comment:
looks like we need "enabled" or "disabled" for `-Dsoup2` rather than "true" or "false":

`../cog-0.16.0/meson.build:1:0: ERROR: Value "false" (of type "string") for combo option "Build with libsoup2 instead of libsoup3" is not one of the choices. Possible choices are (as string): "enabled", "disabled", "auto".`